components/system/components/Typography.js

```diff
@@ -180,12 +163,23 @@ const STYLES_LI = css`
     font-weight: 400;
   }
 
-  a {
-    font-family: ${Constants.font.semiBold};
-    font-weight: 400;
-  }
 `;
 
 export const LI = (props) => {
-  return <li css={STYLES_LI} {...props} />;
+  return <li css={css`${STYLES_LI};${ANCHOR};`} {...props} />;
 };
+
+const ANCHOR = css`
```

Comment:
Interesting idea, but I would prefer this instead:

```js
const ANCHOR = `
  a  {
    font-family: ${Constants.font.text};
    font-weight: 400;
    text-decoration: none;
    color: ${Constants.system.link};
  }

  a:hover {
    text-decoration: underline;
    color: ${Constants.system.link};
  }
`
```

That way you can do:

```js
const STYLES_P = css`
  box-sizing: border-box;
  font-family: ${Constants.font.text};
  font-size: ${Constants.typescale.lvl1};

  ${ANCHOR}
`;
```

And then we only need to:

```js
return <p css={STYLES_P} {...props} />
```